app.js

```diff
@@ -0,0 +1,170 @@
+var express = require('express');
+var path = require('path');
+var fs = require('fs');
+var url = require('url');
+var apiVersion = require('./package').version;
+
+var app = express();
+
+app.set('port', 5000);
+
+app.listen(app.get('port'), function() {
+	console.log('Node app is running on http://localhost:' + app.get('port'));
+});
+
+app.get('/', function(req, res) {
+	var urlParsed = url.parse(req.url, true);
+
+	console.log(urlParsed);
+
+	res.send('<html><body><h1>My web app http API! Version ' + apiVersion + '</h1></body></html>');
+});
+
+app.all('/test/', function(req, res) {
+	res.send('<html><body><h1>Hello test</h1></body></html>');
+});
+
+app.get('/api/' + apiVersion + '/*', function(req, res) {
+	renderDataFromMocks(req, res);
+});
+
+app.delete('/api/' + apiVersion + '/*', function(req, res) {
+
+	deleteDataFromMocks(req, res);
+});
+
+function renderDataFromMocks(req, res) {
+
+	var normalRequestPath = req.path.replace('/' + apiVersion, '') + '/';
+
+	var getFilePath = function(i) {
+		return path.join(
+			__dirname,
+			normalRequestPath,
+			(i / 1000).toFixed(3).replace(/.*\./, ''),
```

Comment:
Nice way. But I'd prefer more general algorithm.

For example list all folders from /users directory and then test using regexp to test if it is integer.
What if you have only `001` and `999`? You will loop unneeded 997 times.

But it is not necessary to implement now, just thoughts.